nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResource.java

```diff
@@ -15,21 +15,37 @@
 import javax.inject.Inject;
 import javax.inject.Named;
 import javax.inject.Singleton;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.Response;
 
 import org.sonatype.nexus.repository.rest.api.AbstractRepositoriesApiResource;
 import org.sonatype.nexus.repository.rest.api.AbstractRepositoryApiRequestToConfigurationConverter;
 import org.sonatype.nexus.repository.rest.api.AuthorizingRepositoryManager;
 import org.sonatype.nexus.repository.rest.api.RepositoriesApiResource;
 import org.sonatype.nexus.rest.Resource;
+import org.sonatype.nexus.validation.Validate;
 import org.sonatype.repository.conan.api.ConanProxyRepositoryApiRequest;
 
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.apache.shiro.authz.annotation.RequiresAuthentication;
+
+import static org.sonatype.nexus.rest.ApiDocConstants.AUTHENTICATION_REQUIRED;
+import static org.sonatype.nexus.rest.ApiDocConstants.INSUFFICIENT_PERMISSIONS;
+import static org.sonatype.nexus.rest.ApiDocConstants.REPOSITORY_CREATED;
+import static org.sonatype.nexus.rest.ApiDocConstants.REPOSITORY_NOT_FOUND;
+import static org.sonatype.nexus.rest.ApiDocConstants.REPOSITORY_UPDATED;
+
 @Named
```

Comment:
you forgot @Api(value = API_REPOSITORY_MANAGEMENT)

Reply:
Fixed 5ec733ed3293c46b3126af127fc30a040e05a962